### PR TITLE
fix an issue where trying to upload large transcoded videos to the cloud would fail

### PR DESCRIPTION
**Pull request by mszulik**

### CODE REVIEW

File: app/Jobs/TranscodeVideo.php [hunk 1]
```diff
@@ -112,13 +105,14 @@ public function failed(?Throwable $exception): void
...
-        $tempDerivativesDirectoryPath = $this->getTempVideoDerivativesDirectoryPath();
         $localDisk = Storage::disk('local');
 
-        MediaStorage::VIDEO_DERIVATIVES->getDisk()->deleteDirectory($tempDerivativesDirectoryPath);
+        MediaStorage::VIDEO_DERIVATIVES->getDisk()->deleteDirectory($this->getTempDerivativesDirectoryPath());
         $localDisk->delete($this->getTempMp4Filename());
         $localDisk->delete($this->getTempOriginalFilename());
         $localDisk->deleteDirectory($this->getFfmpegTempDirectory());
+        // This directory stores local temp derivatives in case cloud storage is used.
+        $localDisk->deleteDirectory($this->version->Media->User->name);
```
Comment (gael-connan-cybex) [inline app/Jobs/TranscodeVideo.php:115]: maybe not kill everything specific to the user that might be running, but something specific to this operation

File: app/Jobs/TranscodeVideo.php [hunk 2]
```diff
@@ -159,6 +153,7 @@ protected function transcodeVideo(): void
 
         $this->localDisk->delete($this->tempOriginalFilename);
         $this->localDisk->deleteDirectory($this->getFfmpegTempDirectory());
+        $this->localDisk->deleteDirectory($this->version->Media->User->name);
```
Comment (gael-connan-cybex) [inline app/Jobs/TranscodeVideo.php:156]: same.

File: app/Jobs/TranscodeVideo.php (hunk 1 above)
Comment (mszulik) [reply app/Jobs/TranscodeVideo.php]: I'm now only deleting the temp derivatives folder for the current version.

File: app/Jobs/TranscodeVideo.php (hunk 2 above)
Comment (mszulik) [reply app/Jobs/TranscodeVideo.php]: I'm now only deleting the temp derivatives folder for the current version.

**Review by gael-connan-cybex — APPROVED** [review]
tested with extra challenges.

**Pull request merged**